styles.css

```diff
@@ -42,4 +41,31 @@ body {
   padding: 0;
   font-family: inherit;
   font-size: 20px;
+  font-weight: bold;
+  color: inherit;
+  border-radius: 0;
+  border: 0;
+  border-left: 1px solid var(--secondaryColor);
+  border-top: 1px solid var(--secondaryColor);
+  cursor: pointer;
+}
+
+.operation {
+  background-color: var(--operationButtonsColor);
+}
+
+.rightBorder {
```

Comment:
This is a bad name for class. You might add other properties to this class and this class won't match the name. Or use a modification from the BEM methodology (e.g. button_right)